backend/ee/onyx/external_permissions/google_drive/group_sync.py

```
@@ -51,7 +51,7 @@ def _get_drive_members(
                 drive_service.permissions().list,
                 list_key="permissions",
                 fileId=drive_id,
-                fields="permissions(emailAddress, type)",
+                fields="permissions(emailAddress, type),nextPageToken",
```

should we throw an exception if this isn't specified for a `execute_paginated_retrieval`?

---

probably yeah